contributors/devel/api-conventions.md

```diff
@@ -595,6 +595,16 @@ Strategic Merge Patch also supports special operations as listed below.
 
 ### List Operations
 
+#### Specifying element ordering for a strategic merge patch on a list [Alpha]
```

Comment:
It is super confusing to put this here. This section of the doc is about when the verb is "list" (i.e., GET on a collection). Please make this part of the strategic merge patch section. Also should we really be editing the doc before it's implemented? I guess I would have expected a proposal.